beacon_node/execution_layer/src/lib.rs

```diff
@@ -1893,26 +1898,42 @@ impl<E: EthSpec> ExecutionLayer<E> {
         &self,
         block_root: Hash256,
         block: &SignedBlindedBeaconBlock<E>,
-    ) -> Result<FullPayloadContents<E>, Error> {
+        spec: &ChainSpec,
+    ) -> Result<SubmitBlindedBlockResponse<E>, Error> {
         debug!(?block_root, "Sending block to builder");
+        if spec.is_fulu_scheduled() {
```

Comment:
Raised a discussion here on whether to switch at the Fulu fork, or when running the node with Fulu scheduled:
https://discord.com/channels/595666850260713488/1395080509775937636/1397431503448969276

IMO it's safer to happen before the fork.